### DISCUSSION TITLE
How to update EDDB reference system with current in-game system using EDDI ?

### DISCUSSION THREAD

**Discussion by shodanx2** [EDCD/EDDI#2334] [EDDI Standalone Q&A] [answered]
It would be great not to have to enter a reference system in EDDB all of the time !
I hope EDDI would auto update the reference system ?

**Comment by Tkael** [comment]
> It would be great not to have to enter a reference system in EDDB all of the time !
> I hope EDDI would auto update the reference system ?

Could you please provide a little more context and elaborate on what you are trying to accomplish?

**Reply by shodanx2** [reply]
EDDB requires a reference system

EDDI knows my current system

How can I make EDDI tell EDDB to use my current system as the reference system for searches?

**Reply by Tkael** [reply]
You can't, at least not directly, since EDDB does not offer an API that EDDI can use to open a search or to define a reference system.

That said,
1. If you use the Inara Responder then as long as you are logged in Inara will know where you are and will be able to use your current system for searches on Inara.
2. If you use VoiceAttack then you can copy the current system name to the clipboard and paste that into EDDB (`{TXT:System name}`).

**Comment by shodanx2** [comment, marked as answer]
Thank you, you are most helpful !

I have not used voice attack yet but that is pretty neat !

The problem with inara is I cannot find planets rich in "Tin" with it, or where nearest place that sells planeraty hangars is.
Is EDDB the only way to find these efficiently ? 

Please mark this question answered, it was not obvious that EDDB lacked this functionality, they even have an accounts system !

**Reply by Tkael** [reply]
For planets with a given material, try https://inara.cz/nearest/#tab_galaxysearchbodies.
For stations with a given module, try https://inara.cz/nearest-outfitting/#tab_galaxysearchoutfitting.
o7
